relational_operators/HashJoinOperator.cpp

```diff
@@ -65,21 +65,11 @@ DEFINE_bool(vector_based_joined_tuple_collector, false,
             "If true, use simple vector-based joined tuple collector in "
             "hash join, with a final sort pass to group joined tuple pairs "
             "by inner block. If false, use unordered_map based collector that "
-            "keeps joined pairs grouped by inner block as they are found "
-            "(this latter option has exhibited performance/scaling problems, "
-            "particularly in NUMA contexts).");
+            "keeps joined pairs grouped by inner block as they are found.");
```

Comment:
My rudimentary profiling also gave the same results. I’ll wait for others to opine. If there are no objections, I’ll remove it.


> On Jun 27, 2016, at 13:57, Harshad Deshmukh <notifications@github.com> wrote:
> 
> In relational_operators/HashJoinOperator.cpp <https://github.com/apache/incubator-quickstep/pull/41#discussion_r68635258>:
> 
> > @@ -65,21 +65,11 @@ DEFINE_bool(vector_based_joined_tuple_collector, false,
> >              "If true, use simple vector-based joined tuple collector in "
> >              "hash join, with a final sort pass to group joined tuple pairs "
> >              "by inner block. If false, use unordered_map based collector that "
> > -            "keeps joined pairs grouped by inner block as they are found "
> > -            "(this latter option has exhibited performance/scaling problems, "
> > -            "particularly in NUMA contexts).");
> > +            "keeps joined pairs grouped by inner block as they are found.");
> I don't mind removing it. If I recall correctly, an older profiling done by @pateljm <https://github.com/pateljm> revealed that map collector outperformed the vector collector pretty much always.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/apache/incubator-quickstep/pull/41/files/258970235f2b2d62cb4faefeffe1d2a8ccd53bc1#r68635258>, or mute the thread <https://github.com/notifications/unsubscribe/ACZB62AXjOwQg0eKFvaerORVzB3rYcSrks5qQB0XgaJpZM4I_RVv>.
>